pkg/imgpkg/cmd/describe.go

```diff
@@ -51,6 +52,7 @@ func NewDescribeCmd(o *DescribeOptions) *cobra.Command {
 	o.RegistryFlags.Set(cmd)
 	cmd.Flags().IntVar(&o.Concurrency, "concurrency", 5, "Concurrency")
 	cmd.Flags().StringVarP(&o.OutputType, "output-type", "o", "text", "Type of output possible values: [text, yaml]")
+	cmd.Flags().BoolVar(&o.IncludeCosignArtifacts, "cosign-artifacts", true, "Do not retrieve cosign artifact information (Default: true)")
```

Comment:
i think the description is flipped. it's including cosign-artifacts by default, right?

Reply:
fixed